src/baldr/graphreader.cc

```diff
@@ -41,17 +41,27 @@ GraphReader::tile_extract_t::tile_extract_t(const boost::property_tree::ptree& p
     try {
       // load the tar
       archive.reset(new midgard::tar(pt.get<std::string>("tile_extract")));
```

Comment:
just realized that the block below is too late, the archive should of course not even get initialized..